src/components/SidePanels/TabPanels/utils.ts

```diff
@@ -210,7 +210,7 @@ export const openSourcePanelDefault: Record<string, PanelBBox> = {
   },
 };
 
-export const partialEmptyBaseProps = {
+export const enterprisePartialEmptyBaseProps = {
```

Comment:
I don't think this makes sense to be qualifying the opensource project like this. Let's revert all of this separation, and just filter it out based on whether comments should be shown or not.

```tsx
panelViews.filter(panel => panel.name !== "comments" || showComments)
```